xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/pvr.h

```diff
@@ -307,6 +307,7 @@ extern "C"
     // Stream demux interface functions
     enum PVR_ERROR(__cdecl* GetStreamProperties)(const struct AddonInstance_PVR*,
                                                  struct PVR_STREAM_PROPERTIES*);
+    void(__cdecl* StreamClosed)(const struct AddonInstance_PVR*);
```

Comment:
Imo this should return `PVR_ERROR` not void, to be able to report for example disconnect errors.

Reply:
> and this function follows the same pattern

Those other stream functions are topic to be moved outside PVR addon API for a long time. When we changed all PVR addon API functions to return PVR_ERROR, @AlwinEsch wanted to leave the stream functions out, because of the intended change and add proper error handling later, when those functions are moved.

> There can be no error


I tend to disagree. What if an implementation of this function needs to contact the PVR backend and this is not available in that very moment, for example?

Reply:
Fair point, I didn’t know the history and this function would not fall into that category. Will add it

Reply:
Done